### ISSUE TITLE
Python version installation issues in Ubuntu 

### ISSUE THREAD

**Issue by Rociocc93** [NKI-CCB/DISCOVER#28]
Hi! 

I was trying to install the python package using conda, but there seems to be a certificate verification issue. See below:

```
Collecting package metadata (current_repodata.json): failed

CondaSSLError: Encountered an SSL error. Most likely a certificate verification issue.

Exception: HTTPSConnectionPool(host='ccb.nki.nl', port=443): Max retries exceeded with url: /software/discover/repos/conda/linux-64/current_repodata.json (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1006)')))
```

Any ideas?

Thank you very much,

Rocío

**Comment by scanisius** [comment]
This seems to be an issue with the SSL certificates of conda. I'll need to look into this for a long-term solution, but for now, there are two things you can do:

- tell conda to use the system-wide certificates of Ubuntu by entering `export REQUESTS_CA_BUNDLE=/usr/lib/ssl/certs/ca-certificates.crt` in your shell before you run the `conda` command (see https://docs.conda.io/projects/conda/en/latest/user-guide/configuration/non-standard-certs.html)
- disable SSL checking altogether by passing the `--insecure` option to the `conda` command, although in general this is not advisable

My recommendation is to go for the first option, which should work if your system has up-to-date certificates.

**Comment by Rociocc93** [comment]
Hello,

Thanks for your quick response! I did try both options, and none of them worked. 

For the first one, I also did the following to make sure that my certificates are up-to-date, but got the exact same error as before:

```
sudo apt-get install --reinstall ca-certificates 
conda config --set ssl_verify /usr/lib/ssl/certs/ca-certificates.crt # to make sure I am reading them from the proper directory
conda config --show ssl_verify
```

For the second option, somehow, the disabling of the SSL checking either with the `--insecure` flag or with `conda config --set ssl_verify false` didn't work either, as I was again getting the same error. The only thing that worked for me was running `export REQUESTS_CA_BUNDE=""` as well, and then after installing, setting everything back to how it was before. 

Thanks again!

Rocío
